roles/horizon/tasks/monitoring.yml

```diff
@@ -6,4 +6,5 @@
 - name: dashboard port check
   sensu_check: name=check-dashboard-clear-port-redirects plugin=check-http.rb
                args='-u http://localhost/ -r'
+  when: haproxy.enabled is defined or haproxy.enabled == True
```

Comment:
When cluster is fronted with a physical/other loadbalancer, there is no need for haproxy